exporter/sumologicexporter/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sumologicexporter
+
+go 1.14
+
+replace go.opentelemetry.io/collector => github.com/SumoLogic/opentelemetry-collector v0.8.0
```

Comment:
Hmm why did you need to add this? I'm almost certain we won't be able to merge this like this to upstream

Reply:
because we are using our modified version of the opentelemetry-collector (in general). I think this exporter doesn't need this. Also I'm going to create more reliable PRs for the upstream